docs/newContributors.md

```diff
@@ -0,0 +1,88 @@
+<h1 align="center">Hello Peeps 👋</h1>
+<h2 align="center">Welcome to the <a href="https://github.com/siv2r/kidney-exchange">kidney-exchange</a> Repo 🎉</h2>
+
+<h2 align="center">Here, is the Complete Guide to Contribute to this Repo.</h2>
+
+<br>
+
+### 1.  Firstly, Join our [Discord](https://discord.com/invite/d7j63WZV82) Server, Introduce yourself there, Interact with Others as 
+> **Open Source is more about Collaboration.** 
+
+<br></br>
+    
+
+### 2.  So to start Contributing to our Project you can solve existing issues or if you have found any bugs or want to add a new feature to Project you can create New Issue, that's purely your choice.
+
+<br>
+
+  ![](../images/Issues.png)
+
+<br></br>
+
+### 3.  Project Maintainers and Mentors will guide you with the issue and incase of any problem you face, you can take help from the community in the discord server.
+ <br></br> 
+
+### 4.  So, Now you have an issue to work upon. So first you need is to fork Repo. "*Fork means producing a Personal Copy of Someone else's Project*". You can fork Repo This Way:-
+
+<br>
+
+  ![](../images/Fork.png)
+
+<br></br>  
+
+### 5.  Clone the Repo you have forked:-
```

Comment:
Could you please remove all the br tags and all the hashes in the numbered lists, thanks!